torch/csrc/lazy/ts_backend/ts_backend_impl.h

```diff
@@ -0,0 +1,50 @@
+#include <torch/csrc/lazy/backend/backend_interface.h>
+
+namespace torch {
+namespace lazy {
+
+class TORCH_API TSData : public torch::lazy::BackendData {
```

Comment:
are all the changes here part of this PR? (or is there any chance you can split the eager changes into their own PR for review?)